python/xorbits/pandas/mars_adapters/core.py

```diff
@@ -203,3 +203,13 @@ def wrapped(*args, **kwargs):
             return from_mars(ret)
 
     return attach_cls_member_docstring(wrapped, member_name, data_type=data_type)
+
+
+class MarsPandasGetItemProxy:
```

Comment:
There's an existing GetItemProxy class `xorbits.numpy.mars_adapters.core.MarsGetItemProxy`. Better move it to `xorbits.core.adapter` so that `xorbits.pandas` don't need to define another GetItemProxy class.

Reply:
Should `DataFrameLoc` inherit from `MarsGetItemProxy ` either? And I didn't find where to attach `df.loc`'s doc, is it missing or somewhere.